### ISSUE TITLE
Why "npm unlink x" is an uninstall alias?

### ISSUE THREAD

**Issue by davidcosama** [npm/npm#19973]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [X] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

Why "npm unlink x" is an uninstall when "npm link x" only creates a symbolic link? Would not be better to undo the symbolic link?

Maybe, it can be a philosophical question, but I should do it.

Thanks in advance!

### supporting information:

 - `npm -v` prints:
 - `node -v` prints:
 - `npm config get registry` prints:
 - Windows, OS X/macOS, or Linux?:
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by tommck** [comment]
Yeah.. it actually removes it from package.json which makes NO sense to me
